python/core/auto_generated/layertree/qgslayertreemodellegendnode.sip.in

```diff
@@ -94,6 +94,14 @@ Entry point called from QgsLegendRenderer to do the rendering.
 Default implementation calls drawSymbol() and drawSymbolText() methods.
 
 If ctx is null, this is just first stage when preparing layout - without actual rendering.
+%End
+
+    void draw( const QgsLegendSettings &settings, QJsonObject &json );
+%Docstring
+Entry point called from QgsLegendRenderer to do the rendering in a \a
+json object.
```

Comment:
json -> JSON (in dox comments, not class/variable name)

Reply:
done